### ISSUE TITLE
 how to gradient descent

### ISSUE THREAD

**Issue by shz1314** [ZZUTK/Face-Aging-CAAE#19]
hi @ZZUTK 
  i have a problem about your model's training.first i remove the GAN loss,just use autoencorder to generate blurred image.Then set other weight of loss from 0 to 0.0001 ,but the EG_err=0.2870 at 50th epoch.
![2017-11-05 22-37-02](https://user-images.githubusercontent.com/18184299/32415775-e77c85e6-c279-11e7-8791-efb67b9b3878.png)

thanks
Song

**Comment by susanqq** [comment]
Can you try to set the stddev for weight init as 0.02?

**Comment by shz1314** [comment]
Thanks @susanqq 
  can you tell me where should i add it, the stddev for weight.
 so thanks,
Song

**Comment by shz1314** [comment]
OH i find ,do you mean the weight of 'fc' 'conv''deconv'

**Comment by susanqq** [comment]
yes, please change it in the ops.py

**Comment by shz1314** [comment]
hi,
   shall you send me the pretrained model to me, i want to do the contrast experiment with your model.My email is haoze_song_jnu@163.com .
Yours Haoze
